pkg/vm/engine/disttae/engine.go

```diff
@@ -267,6 +267,10 @@ func (e *Engine) GetRelationById(ctx context.Context, op client.TxnOperator, tab
 	return
 }
 
+func (e *Engine) AllocateID(ctx context.Context) (uint64, error) {
+	return e.idGen.AllocateID(ctx)
```

Comment:
You shouldn't use this. This is for assigning table ids

Reply:
Index metadata tables require an ID that is of the same type as tableId and databaseID